### ISSUE TITLE
[Feature] `vela env init` should only create namespace if not existing

### ISSUE THREAD

**Issue by jriguera** [oam-dev/kubevela#1157]
**Is your feature request related to a problem? Please describe.**

We want to manage user using RBAC, each user would have access to a pre-created namespace (but they do not have access to create new ones), when we use `vela env init --namespace TEAM  TEAM`, vela cli fails with:

```
Error: namespaces is forbidden: User "user@account.com" cannot create resource "namespaces" in API group "" at the cluster scope: requires one of ["container.namespaces.create"] permission(s).
```

**Describe the solution you'd like**
Kubevela should check if the namespace is already existing and if so try to get it and not created (the user has `container.namespaces.get` access).

**Describe alternatives you've considered**
We have created this workaround instead of using `vela env init`:
```
TEAM="my-team"
mkdir -p ~/.vela/envs/$TEAM
echo {"name":"$TEAM","namespace":"$TEAM","domain":"","issuer":""}  > ~/.vela/envs/$TEAM/config.json
```
But we would like a more user friendly solution with `vela env init`.

**Additional context**
This is on GKE with RBAC and role bindings.

**Comment by wonderflow** [comment]
Yes, thanks for reporting

**Comment by wonderflow** [comment]
@mosesyou Hi, are you working on that? I can fix along with #1174

**Comment by mosesyou** [comment]
> @mosesyou Hi, are you working on that? I can fix along with #1174

Sry, because something delayed doing this. Have you completed it? If not I will fix it in a while.

**Comment by wonderflow** [comment]
@mosesyou not yet, please go ahead
